src/index.js

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
При импорте либ обычно `node_modules` не надо, должно заресолвится само
```suggestion
import 'bootstrap/dist/css/bootstrap.min.css';
```